MetamodelAlgemeen.md

```diff
@@ -1669,7 +1669,23 @@ Hier kan in tekst een beschrijving opgenomen worden over de kwaliteit van de inw
 
 *Toepassing:* Objecttype
 
+#### Metagegeven: **Eenheid**
```

Comment:
Toegevoegde tekst gaf merge conflict, dat heb ik opgelost en onderstaande wijzigingen doorgevoerd. Dat heb ik eerst gedaan en toen zag ik dat er nog om een review gevraagd is, dus mijn voorstellen staan nu al in de pullrequest verwerkt